src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
+@import "./utils/reset.scss";
+@import "./blocks/container.scss";
+@import "./blocks/block.scss";
```

Comment:
if you make the blocks elements of the container, do not forget to remove the file